### DISCUSSION TITLE
Account theme

### DISCUSSION THREAD

**Discussion by LVasseurIppon** [keycloakify/keycloakify#180] [General]
Hello,

Is there any possibility to use Keycloakify to custom the account theme ?

**Comment by garronej** [comment]
Not yet sorry 😕

https://docs.keycloakify.dev/limitations#login-and-email-only

**Comment by bekihm** [comment]
@garronej Is there a plan to support it in future?

**Reply by garronej** [reply]
Yes, a release candidate for the v7 has been published tonight.  
V7 features accout theme support
It isn't documented yet thoug

**Reply by bekihm** [reply]
Very nice :-) thank you

**Comment by garronej** [comment]
It's now fully featured.  

https://youtu.be/WMyGZNHQkjU?t=203
